### PR TITLE
Included a script to start a mock search space service and HPO REST API service

### PR DESCRIPTION
**Pull request by chandrams**
Included a script to start a mock search space service and HPO REST API service

Signed-off-by: Chandrakala Subramanyam <csubrama@redhat.com>

### CODE REVIEW

File: tests/scripts/start_hpo_servers.sh [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+	[?])
+		usage
+	esac
+done
+
+if [ -z "${log_dir}" ]; then
+        log_dir="${HOME}"
+fi
+
+
```
Comment (shruacha1234) [inline tests/scripts/start_hpo_servers.sh:54]: Remove extra space

File: tests/scripts/start_hpo_servers.sh [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+function usage() {
+	echo ""
+	echo "Usage: $0 -p log_dir -j <search space json> [-s] [-t]" 
+	echo "Where -s = start(default), -t = terminate" 
+	echo ""
+	exit -1
+
```
Comment (shruacha1234) [inline tests/scripts/start_hpo_servers.sh:61]: Remove extra space

File: tests/scripts/start_hpo_servers.sh [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+##### Script to start the HPO (Hyperparameter Optimization) REST Service and Search Space Mock server required for testing HPO REST APIs #####
+
+CURRENT_DIR="$(dirname "$(realpath "$0")")"
+SCRIPTS_DIR="${CURRENT_DIR}"
+
+PATH_TO_PYTHON_SERVICE=${SCRIPTS_DIR}/../../hyperparameter_tuning
```
Comment (shruacha1234) [inline tests/scripts/start_hpo_servers.sh:23]: Change python to HPO

File: tests/scripts/start_hpo_servers.sh [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+	export N_JOBS=1
+	export PYTHONUNBUFFERED=TRUE
+
+	# Start the Search Space service
+	echo "Starting searchspace service..."
+	nohup python3 ${PATH_TO_SEARCHSPACE}/search_space.py ${SEARCH_SPACE_JSON} &> ${log_dir}/searchspace.log &
+
+	# Start the HPO REST API service 
+	echo "Starting HPO REST API service..."
+	nohup python3 ${PATH_TO_PYTHON_SERVICE}/service.py  &> ${log_dir}/service.log &
```
Comment (shruacha1234) [inline tests/scripts/start_hpo_servers.sh:75]: change PYTHON to HPO

**Review by shruacha1234 — APPROVED** [review]
LGTM

**Review by dinogun — APPROVED** [review]
LGTM. As mentioned we need an updated README to indicate how these tests can be invoked. Approving this assuming it will be a separate PR.

**Pull request merged**